.github/workflows/branch-cleanup.yml

```diff
@@ -28,7 +28,6 @@ jobs:
       - uses: actions/github-script@v7
         env:
           REPO: ${{ matrix.repo }}
-          DRY_RUN: true
```

Comment:
It might make sense to define this as an input, so we can switch it on/off using an org-level (or repo-level) variable 🤔